src/components/Home/Home.js

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react';
+import './Home.css';
+import TipCalc from '../TipCalc/TipCalc';
+import BillSplit from '../BillSplit/BillSplit';
```

Comment:
Could you make TipCalc and BillSplit into one application? I think that from an UX perspective might make the website more user friendly.